examples/all-sky/rrtmgp_allsky.F90

```diff
@@ -338,6 +339,9 @@ program rte_rrtmgp_clouds
   !
   !!$omp parallel do firstprivate(fluxes)
   do iloop = 1, nloops
+    ! Omit the checks starting with the second iteration
+    if (iloop == 2) call rte_config_checks(logical(.false., wl))
```

Comment:
@alexeedm Mmmm, do we want to make this decision on the user's behalf?

Reply:
This is an example, so all the loops are identical. If the checks pass on the first loop, they do pass on all the rest

Reply:
True enough... I guess you're suggesting that timing with the checks off is more useful information than timing with the checks on? Maybe we should note this somewhere (in the output? in the Readme?)

Reply:
Yep, without the checks we can do timings pretty accurately in this example. Your call regarding the note!